scheduler/src/cook/components.clj

```diff
@@ -61,15 +62,28 @@
                 [:headers "Cache-control"]
                 "max-age=0"))))
 
+(def rate-limiters
+  {:create-job-rate-limiter
+   (fnk [[:settings [:rate-limit expire-minutes job-creation]]]
+     (if job-creation
+       (let [{:keys [bucket-size is-enforcing rate-per-minute]} job-creation]
+         (when-not (or (true? is-enforcing) (false? is-enforcing))
+           (throw (ex-info ":is-enforcing not a boolean " job-creation)))
+         ((util/lazy-load-var 'ratelimit/tbf-ratelimiter-make) bucket-size rate-per-minute expire-minutes is-enforcing))
```

Comment:
Why are `bucket-size`, `rate-per-minute` and `expire-minutes` not validated here? I think we should group the validation together in a single place.

Reply:
They're validated in the ratelimit code.